lib/parse/config/parseAccentColorParam.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ *
+ * @param param Querystring param value.
+ * @returns Array of HEX CSS color strings. Can contain syntax for color location in gradient colors.
+ */
+const parseAccentColorParam = (param: string | string[]): string[] => {
+  const colors = (Array.isArray(param) ? param : [param])
+    .map((ac) =>
+      /^(?:[a-f0-9]{2}[a-f0-9]{2}[a-f0-9]{2}(?:[a-f0-9]{2})?|[a-f0-9]{3})(?:\s1?\d?\d%)?$/i.test(
```

Comment:
This will match HEX color strings supported by CSS, 3-char (`FFF`) and 6-char (`FFFFFF`) and 8-char (w/ alpha) (`FFFFFFFF`). It also allows for a percentage following a space. This is used to position colors in the gradient created when there are multiple accent colors provided.